README.md

```
@@ -34,4 +34,4 @@ docker-compose -f docker-compose.prod.yml down
 
 # Seed database
 ## In Development:
-curl -X POST http://localhost:3000/api/seed-default-stories -H "Content-Type: application/json" -d '{}'
\ No newline at end of file
+curl -X POST http://localhost:3000/api/seed-default-stories -H "Content-Type: application/json" -d '{}'
```

markdownlint-cli2 (0.17.2)

Bare URL used

(MD034, no-bare-urls)